### PR TITLE
Creating plots and getting characteristics vignettes using Sphinx documentation

### PR DESCRIPTION
**Pull request by aefitts**
Two vignettes containing information on creating plots and getting watershed characteristics using Sphinx documentation. @nkorinek

### CODE REVIEW

File: examples/plot_get_characteristics.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+lat, lon = 39.966256, -105.482227
+ws = streamstats.Watershed(lat=lat, lon=lon)
+
+###########################################################################
+# Find the Hydrologic Unit Code (HUC) of the watershed
+# -----------------------------------------------------
+# The USGS delineates watershed using a series of numbers based a hierarchal
+# region system. Every watershed is assigned a series of numbers called the
+# hydrological unit code (HUC). StreamStats uses HUC to identify and delineate
```
Comment (nkorinek) [inline examples/plot_get_characteristics.py:32]: Maybe link to some USGS docs on how they assign HUCs here?

File: examples/plot_get_characteristics.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# watersheds. The ``ws.huc`` function will return the HUC of the identified
+# watershed.
+
+ws.huc
+
+###########################################################################
+# Find Characteristics of the watershed
+# --------------------------------------
+# The function ``ws.characteristics`` will return the available basin
```
Comment (nkorinek) [inline examples/plot_get_characteristics.py:41]: Since this is a function, would probably help to put parenthesis after it!

**Review by nkorinek — CHANGES_REQUESTED** [review]
Great job overall! Some minor notes on wording and specifics, but really well done in general! If you disagree with any of my notes just let me know and we can talk about it, happy to here your thoughts!

**Comment by nkorinek** [comment]
@aefitts review above! Good job overall!

**Comment by nkorinek** [comment]
@mbjoseph Hey! Ally and I worked on your docs and made a PR updating them to add sphinx gallery and some vignettes. If you want to review the PR, that would be awesome! But if you don't have time to be actively maintaining this repo no worries either, we would just need a way to merge it without you having to review it. Either way, let us know, thanks!

**Comment by mbjoseph** [comment]
@nkorinek thanks for the heads up -- I'll leave it to you and the Earth Lab team to review etc. Great to see this come to fruition! :clap:

**Pull request merged**